sql/expression/set.go

```diff
@@ -0,0 +1,54 @@
+package expression
+
+import (
+	"fmt"
+	"github.com/src-d/go-mysql-server/sql"
+	"gopkg.in/src-d/go-errors.v1"
+)
+
+var errCannotSetField = errors.NewKind("Expected GetField expression on left and but %T")
+
+// Set updates the value of a field from a row.
+type Set struct {
```

Comment:
Would call this SetField (Set is too overloaded). Or SetColumn if Column is used over Field in other code.